### PR TITLE
sortbootorder.c: merge all USB entries

### PR DESCRIPTION
**Pull request by macpijan**

### CODE REVIEW

File: sortbootorder.c [hunk 1]
```diff
@@ -164,7 +164,12 @@ int main(void) {
 				break;
 			default:
 				if (key >= 'a' && key <= 'm' ) {
-					move_boot_list( bootlist, get_line_number(max_lines,key), max_lines );
+					u8 line_start = 0;
```
Comment (pietrushnic) [inline sortbootorder.c]: @macpijan IIRC most of sortbootorder code follow C89 standard, which requires variable declaration on top of the function. We should keep code style, if there is no important reason for not doing so. Please move this local declarations to top of the function.

File: sortbootorder.c [hunk 2]
```diff
@@ -189,18 +194,18 @@ static int strcmp_printable_char(const char *s1, const char *s2)
...
 			break;
 	}
-	return (i == line_cnt) ? 0 : i;
+	return (i == line_start - 1) ? 0 : i;
 }
 
 /*******************************************************************************/
 static void copy_list_line( char *src, char *dest ) {
-u8 i=0;
+int i=0;
```
Comment (pietrushnic) [inline sortbootorder.c]: @macpijan what is the reason behind this change ?

File: sortbootorder.c [hunk 3]
```diff
@@ -223,10 +228,17 @@ static void show_boot_device_list( char buffer[MAX_DEVICES][MAX_LENGTH], u8 line
...
 			if (strcmp_printable_char(&(buffer[i][0]), &(bootlist_def[y][0])) == 0) {
-				strcpy(print_device, &bootlist_map[y][0]);
-				print_device[strlen(print_device)-1] = '\0';
-				printf("  %s %s\n", print_device, (device_toggle[y]) ? "" : "(disabled)");
-				break;
+				unique = 1;
+				for (j = 0; j < y; j++) {
+					if (strcmp_printable_char(&bootlist_map[y][0], &bootlist_map[j][0])  ==  0)
+						unique = 0;
+				}
```
Comment (pietrushnic) [inline sortbootorder.c]: @macpijan why we need this ?

File: sortbootorder.c (hunk 2 above)
Comment (macpijan) [inline sortbootorder.c]: No reason, I overlooked this - this shouldn't have been commited

File: sortbootorder.c (hunk 3 above)
Comment (macpijan) [inline sortbootorder.c]: As far as I know, there still have to be four separate entries in bootorder files. If we want to merge USB options into one in sortbootorder menu then we have to set the same name to all of them and print only one of them. This prevents from printing several items with the same name.
